SMLHelper/MonoBehaviours/CustomCreature.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SMLHelper.V2.MonoBehaviours
+{
+    using UnityEngine;
+
+    public class CustomCreature : MonoBehaviour
```

Comment:
If the class is _not_ intended to be manipulated by other assemblies, make it `internal`.